patreon/api.py

```diff
@@ -16,27 +15,28 @@ def __init__(self, access_token):
 
     def fetch_user(self, includes=None, fields=None):
         return self.__get_jsonapi_doc(
-            build_url('current_user', includes=includes, fields=fields)
+            build_url('identity', includes=includes, fields=fields)
         )
 
-    def fetch_campaign_and_patrons(self, includes=None, fields=None):
-        if not includes:
-            includes = campaign.default_relationships \
-                + [campaign.Relationships.pledges]
+    def fetch_campaigns(self, includes=None, fields=None):
         return self.fetch_campaign(includes=includes, fields=fields)
 
-    def fetch_campaign(self, includes=None, fields=None):
+    def fetch_campaigns(self, includes=None, fields=None):
         return self.__get_jsonapi_doc(
             build_url(
-                'current_user/campaigns', includes=includes, fields=fields
+                'campaigns', includes=includes, fields=fields
             )
         )
 
-    def fetch_page_of_pledges(
-            self, campaign_id, page_size, cursor=None, includes=None,
-            fields=None
-    ):
-        url = 'campaigns/{0}/pledges'.format(campaign_id)
+    def fetch_campaign(self, campaign_id, includes=None, fields=None):
+        return self.__get_jsonapi_doc(
+            build_url(
+                'campaigns/{0}'.format(campaign_id), includes=includes, fields=fields
+            )
+        )
+
+    def fetch_members(self, campaign_id, page_size, cursor=None, includes=None, fields=None):
+        url = url = 'campaigns/{0}/members'.format(campaign_id)
         params = {'page[count]': page_size}
         if cursor:
```

Comment:
this code assumes `cursor` will be a datetime object -- is that still the case for paginating through members? https://docs.patreon.com/#get-api-oauth2-v2-campaigns-campaign_id-members implies it is not

Reply:
I don't think we changed the cursor logic at all for APIv2, so we should be fine?